File: Core/DTO/Car.cs
namespace Core.DTO
{
	public class CarDTO
	{
		public int Id { get; set; }
		public string NumberPlate { get; set; }
		public string Color { get; set; }
		public string CarTypeId { get; set; }
		public string CarTypeName { get; set; }
		public decimal Cost { get; set; }
		public byte Seat { get; set; }
		public string ImgPath { get; set; }
	}
}

File: WebApp/ClientApp/src/types/Car.js
class Car {
	constructor(id = 0, numberPlate = "", carTypeName = "", imgPath = "", cost = 0, seat = 0, color = "") {
		this.id = id;
		this.numberPlate = numberPlate;
		this.carTypeName = carTypeName;
		this.imgPath = imgPath;
		this.cost = cost;
		this.seat = seat;
		this.color = color;
	}
}

export default Car;